client/src/modules/vouchers/toolkit/patient_fled/patient_fled.js

```diff
@@ -0,0 +1,192 @@
+angular.module('bhima.controllers')
+  .controller('PatientFledKitController', PatientFledKitController);
+
+PatientFledKitController.$inject = [
+  '$uibModalInstance', 'DebtorGroupService', 'NotifyService',
+  'SessionService', 'bhConstants', '$translate', 'VoucherToolkitService',
+];
+
+// Import transaction rows for a convention payment
+function PatientFledKitController(
+  Instance, DebtorGroup, Notify, Session, bhConstants, $translate, ToolKits,
+) {
+  const vm = this;
+
+  const { MAX_DECIMAL_PRECISION } = bhConstants.precision;
+
+  // global variables
+  vm.debtorGroupFilter = { is_convention : 1 };
+  vm.gridOptions = {};
+  vm.account = {};
+  vm.enterprise = Session.enterprise;
+
+  // expose to the view
+  vm.selectGroupInvoices = selectGroupInvoices;
+  vm.close = Instance.close;
+  vm.import = submit;
+
+  // Set up page elements data (debtor select data)
+  vm.onSetlectPatient = onSetlectPatient;
+  vm.onSelectAccount = onSelectAccount;
+
+  function onSetlectPatient(patient) {
+    vm.patient = patient;
+    selectGroupInvoices(vm.patient);
+  }
+
+  function onSelectAccount(account) {
+    vm.account.id = account.id;
+  }
+
+
+  // get debtor group invoices
+  function selectGroupInvoices(patient) {
+    vm.loading = true;
+
+    DebtorGroup.invoices(patient.debtor_uuid, { debtorByuuid : 1 })
```

Comment:
It looks like we already have this functionality in the [DebtorService](https://github.com/IMA-WorldHealth/bhima/blob/master/client/src/modules/debtors/debtor.service.js#L44).  Can you just use that instead of re-implementing it here?